assets/js/admin.js

```diff
@@ -50,6 +60,39 @@
             $('.tab-' + which).addClass('active');
         });
 
+        // Loads more logs when 'Load More' button is clicked.
+        $(document).on('click', '.load-more', function() {
+
+          var data = {
+            action: 'postmark_load_more_logs',
+            offset: logs_offset,
+            _wpnonce : $('#_wpnonce').val()
+          };
+
+          // Prepares for next 'Load More' by increasing the offset amount for next query.
+          logs_offset += 10;
+
+          $.post( ajaxurl, data, function( response ) {
+
+            // Parses response string into JSON.
+            response = JSON.parse(response);
+
+            // Adds new logs to logs table.
+            $('#pm-log-table tr:last').after(response.html);
```

Comment:
I'm not sure if this is the wordpress recommended way of rendering this content. Usually, we would want to look at something like using a simple client-side template, but that might be overkill here.

Reply:
Also, if the logs themselves contain data that is user-submitted (email addresses), these should be run through some sort of html-encoder on the server side before being rendered by the client. This is to prevent XSS attacks from malicious email-address submitters.

Reply:
Added sanitation for these fields before insertion into logs db.